pkg/network/apis/network/validation/validation.go

```diff
@@ -16,6 +16,28 @@ import (
 	"github.com/openshift/origin/pkg/util/netutils"
 )
 
+func validateCIDRv4(cidr string) (*net.IPNet, error) {
+	ipnet, err := netutils.ParseCIDRMask(cidr)
+	if err != nil {
+		return nil, err
+	}
+	if ipnet.IP.To4() == nil {
+		return nil, fmt.Errorf("must be an IPv4 network: %s", cidr)
+	}
+	return ipnet, nil
+}
+
+func validateIPv4(ip string) (net.IP, error) {
+	bytes := net.ParseIP(ip)
+	if bytes == nil {
+		return nil, fmt.Errorf("invalid IP address: %s", ip)
+	}
+	if bytes.To4() == nil {
+		return nil, fmt.Errorf("must be an IPv4 address: %s", ip)
```

Comment:
Don't include the value in the error message, it's already included in the validation output.